csatar-plugins/csatar/models/scout/fields.yaml

```diff
@@ -29,3 +29,38 @@ fields:
         label: Gender
         span: auto
         type: number
+    is_active:
+        label: Active
+        span: auto
+        default: 1
+        type: checkbox
+    legal_relationship:
+        label: 'Legal Relationship'
+        nameFrom: title
+        descriptionFrom: description
+        span: auto
+        type: relation
+    chronic_illnesses:
+        label: 'Chronic Illnesses'
+        nameFrom: title
+        descriptionFrom: description
+        span: auto
+        type: relation
+    special_diet:
+        label: 'Special Diet'
+        nameFrom: title
+        descriptionFrom: description
+        span: auto
+        type: relation
+    religion:
+        label: Religion
+        nameFrom: title
+        descriptionFrom: description
+        span: auto
+        type: relation
+    tshirt_size:
+        label: 'T-Shirt Size'
+        nameFrom: title
+        descriptionFrom: description
+        span: auto
+        type: relation
```

Comment:
Please add 'empty option' every drop-down, even if it's mandatory, to avoid auto accidental auto selection...